tests/unit/commands/local/lib/test_local_lambda.py

```diff
@@ -625,6 +625,45 @@ def test_must_raise_if_imageuri_not_found(self):
             self.local_lambda.invoke(name, event, stdout, stderr)
 
 
+class TestLocalLambda_invoke_with_container_host_option(TestCase):
+    def setUp(self):
+        self.runtime_mock = Mock()
+        self.function_provider_mock = Mock()
+        self.cwd = "/my/current/working/directory"
+        self.debug_context = None
+        self.aws_profile = "myprofile"
+        self.aws_region = "region"
+        self.env_vars_values = {}
+        self.container_host = "localhost"
```

Comment:
This should be testing a different value from the default.

Reply:
The default value of `container_host` is `None` in this class. Since it's tested in isolation, the default value of the option won't be passed into here.